app/src/main/java/com/jerboa/ui/components/common/AppBars.kt

```diff
@@ -670,7 +670,7 @@ fun JerboaPullRefreshIndicator(
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
-fun SaveTopBar(
+fun ActionTopBar(
     onBackClick: () -> Unit,
     onSaveClick: () -> Unit,
```

Comment:
Change `onSaveClick`, `saveText`, `saveIcon` to

`onSubmitClick`, `submitText`, `submitIcon`, etc. These aren't always saves, but are often creates.